app/presenters/menu/default_menu.rb

```diff
@@ -23,12 +23,19 @@ def cloud_inteligence_menu_section
           Menu::Item.new('dashboard',  N_('Dashboard'),  'dashboard',  {:feature => 'dashboard_view'},           '/dashboard/'),
           Menu::Item.new('report',     N_('Reports'),    'miq_report', {:feature => 'miq_report', :any => true}, '/report/explorer'),
           # Menu::Item.new('usage',    N_('Usage'),      'usage',      {:feature => 'usage'},                    '/report/usage/'), #  / Hiding usage for now - release 5.2
+          consumption_menu_section,
           Menu::Item.new('chargeback', N_('Chargeback'), 'chargeback', {:feature => 'chargeback', :any => true}, '/chargeback/explorer'),
           Menu::Item.new('timeline',   N_('Timelines'),  'timeline',   {:feature => 'timeline'},                 '/dashboard/timeline/'),
           Menu::Item.new('rss',        N_('RSS'),        'rss',        {:feature => 'rss'},                      '/alert/')
         ])
       end
 
+      def consumption_menu_section
+        Menu::Section.new(:cons, N_("Consumption"), 'fa fa-plus fa-2x', [
+          Menu::Item.new('consumption', N_('Dashboard'), 'consumption', {:feature => 'consumption', :any => true}, '/consumption/show')
```

Comment:
Rubocop throw an error here , it seems like the others functions in the file like services_menu_section or clouds_menu_section. Is it necesary fix this?

app/presenters/menu/default_menu.rb

:large_orange_diamond: - Line 35, Col 11 - Style/IndentArray - Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.
:small_blue_diamond: - Line 35, Col 121 - Metrics/LineLength - Line is too long. [135/120]